xrpl/utils/tx_parser/orderbook_changes.py

```diff
@@ -0,0 +1,74 @@
+"""Parse order book changes caused by a transaction."""
+
+from typing import Any, Dict, List, Union
+
+from xrpl.utils.tx_parser.utils import (
+    group_by_address_order_book,
+    is_valid_metadata,
+    normalize_nodes,
+    parse_order_book_changes,
+)
+
+
+class OrderBookChanges:
+    """Parse order book changes from a transactions metadata
+    in a easy-to-read format.
+
+    Args:
+        metadata (`Dict[str, Union[str, int, bool, Dict[str, Any]]]`):
+            The transactions metadata.
+    """
+
+    all_orderbook_changes: Dict[str, List[Dict[str, Any]]] = {}
+    """All orderbook_changes"""
```

Comment:
What does the key represent?

Reply:
It represents the accounts address :)

Reply:
Could you add that to the docstring for future reference?